xml/cap_admin_nproc_limits.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.manage-nproc-limits"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Managing nproc limits of pods</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  In &productname; there are parameters,
+  <command>kube.limits.nproc.soft</command> and
+  <command>kube.limits.nproc.hard</command>, to configure a soft nproc limit
+  and a hard nproc limit for <literal>scf</literal> pods. By default, these
```

Comment:
Maybe add a brief definition of nproc

Reply:
Updated in https://github.com/SUSE/doc-cap/pull/414/commits/0e6e35aeffbbf7c98e87eec1312c3ffc5822f6bd